frontend/src/app/modules/home/home.module.ts

```diff
@@ -1,18 +1,28 @@
 import { NgModule } from '@angular/core';
 import { SharedModule } from '@shared/shared.module';
+import { ReactiveFormsModule } from '@angular/forms';
 
+import { PrimeComponentsModule } from '@shared/modules/prime-components/prime-components.module';
 import { HomeComponent } from './home-page/home-page.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
 import { HomeRoutingModule } from './home-routing.module';
+import { ProjectsComponent } from './projects/projects.component';
+import { IssuesComponent } from './issues/issues.component';
+import { AddDashboardComponent } from './add-dashboard/add-dashboard.component';
 
 @NgModule({
     declarations: [
         HomeComponent,
         DashboardComponent,
+        ProjectsComponent,
+        IssuesComponent,
+        AddDashboardComponent,
     ],
     imports: [
         SharedModule,
-        HomeRoutingModule
+        HomeRoutingModule,
+        PrimeComponentsModule,
+        ReactiveFormsModule,
```

Comment:
They are exported via SharedModule already. So, you don't need do that again

Reply:
Ok, thanks